src/librehatti/settings.py

```diff
@@ -53,7 +53,7 @@
 STATIC_URL = '/static/'
 
 STATICFILES_DIRS = (
-    '/path/to/LibreHatti/static',
+    '/home/amarjeet/projects/LibreHatti/static',
 )
 
```

Comment:
You are not supposed to change settings file. This PR is closed. You should send new one.